ext/pdo_pgsql/pdo_pgsql.stub.php

```diff
@@ -52,4 +52,6 @@ public function lobUnlink(string $oid): bool {}
     public function getNotify(int $fetchMode = PDO::FETCH_DEFAULT, int $timeoutMilliseconds = 0): array|false {}
 
     public function getPid(): int {}
+
+    public function setNoticeCallback(?callable $callback): void {}
```

Comment:
The way I would implement this, is by adding an annotation above this: `/** @implementation-alias PDO_PGSql_Ext::pgsqlSetNoticeCallback */`. This way, you don't have to change the C code at all.

Reply:
Every one of your comments is a discovery for me…
I'll try it, let's wait for the CI.

Still, I aligned the .c to other internal methods (removing the redundant `proto` that still mentionned the `mixed` callback from PHP 5 time, instead of the new `?callable`)